docs/embedding/embedded-analytics-js.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Embedded Analytics JS
+summary: Getting started with Embedded Analytics JS for embedding Metabase entities into external applications
+---
+
+Embedded analytics JS allows you to embed Metabase entities like questions, dashboards, or even the query builder into your own application using customizable components.
+
+Embedded Analytics JS is a JavaScript library built on top of Metabase's [Embedded Analytics React SDK](./sdk/introduction.md). But it does not require using React or setting up full SDK embedding.
+Unlike with [interactive embedding](./interactive-embedding.md), where you embed the entire Metabase app in an iframe, Embedded Analytics JS lets you choose from a set of predefined components like a single chart, a dashboard with optional drill-through, or query builder, and customize those components.
+
+Embedded Analytics JS leverages [JWT SSO](../people-and-groups/authenticating-with-jwt.md) to authenticate people and automatically apply permissions to show people the right data upon sign-in.
+
+Currently you can choose to embed:
+
+- A dashboard
+- A question (chart). You can embed both questions built with the query builder and questions built with SQL.
+- Full graphical [query builder](../questions/query-builder/editor.md) to enable people to build their own charts and explorations.
+
+## Quickstart
+
+### 1. Enable Embedded Analytics JS
+
+1. In Metabase, go to **Admin Settings > Embedding > Modular embedding**.
+2. Toggle on **Embedded Analytics JS**.
+3. Under **Cross-Origin Resource Sharing (CORS)**, add the URLs of the websites where you want to embed Metabase (such as `https://*.example.com`). For testing embeds, you can use `localhost` which is always included in CORS policy.
+
+### 2. Create a new embed
+
+1. In Metabase, click on **+ New** button in top right corner and select **Embed**.
```

Comment:
```suggestion
1. In Metabase, click on **+ New** button in top right corner and select **Embed**. Note that this will only be visible to admins.
```